### ISSUE TITLE
Disable snow on prefers-reduced-motion

### ISSUE THREAD

**Issue by Maingron** [Maingron/snow.js#1]
DIsable the script / Don't even run it when the users requests reduced motion.
This can help people that get nausea or anxious from the snow, especially from the jittering.

This should be possible through CSS `@media screen and (prefers-reduced-motion: reduce) { }` or maybe even through Javascript. If it's possible to get three values (no motion; less motion; normal), maybe disable the jitter on less motion and disable the snow entirely on no motion.

**Comment by Maingron** [comment]
Added in [Version 1.2.0](https://github.com/Maingron/snow.js/releases/tag/1.2.0). Also added a notice in example.html.
